app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,12 @@
+<resources>
+
+    <string name="app_name">YouTube AdAway</string>
+    <string name="module_active">¡está activo!</string>
+    <string name="module_inactive">NO está activo!\nAsegúrese de que esté habilitado y REINICIE.</string>
+    <string name="donate">Donar</string>
+    <string name="no_thanks">No, gracias</string>
+    <string name="whats_in">"Que hay:\nBloquear anuncios en video\nHabilitar reproducción en segundo plano\nBloquear tarjetas publicitarias (BETA)"</string>
```

Comment:
I do not like how "Que hay" sounds (I think it could be a confusing way to word this), but since other translations use far more letters, what about something like "Características" (Features)?

```suggestion
    <string name="whats_in">"Características:\nBloquea anuncios en video\nHabilita reproducción en segundo plano\nBloquea tarjetas publicitarias (BETA)"</string>
```

In case sticking to the original words is preferred, there is a typo in "Qué":

```suggestion
    <string name="whats_in">"Qué hay:\nBloquea anuncios en video\nHabilita reproducción en segundo plano\nBloquea tarjetas publicitarias (BETA)"</string>
```

I also think third person sounds better than infinitives. Second person also sounds better to me than infinitives. Talking about second person, I think "tú" is a bit more general than "usted" and texts would sound "friendlier" to people if they used "tú" (disclaimer: I am from a region where "usted" is used, so not a personal preference).